lua/dependency_assist/rust/crates_api.lua

```diff
@@ -15,7 +15,7 @@ function M.search_package(name, cb)
       local helpers = require("dependency_assist/utils/helpers")
       helpers.echoerr(name .. "not found")
     end
-  end)
+  end, '"User-Agent: my_bot (my_bot.com/info)"')
```

Comment:
I don't really use the rust side of this plugin at all tbh, do you know why this change has happened/is needed.
Also, why `my_bot` if it's arbitrary can it be a more identifiable name?